packages/transport/.eslintrc.js

```diff
@@ -2,11 +2,10 @@ module.exports = {
     rules: {
         // todo: remove when refactoring transport layer.
         'no-underscore-dangle': 'off',
-        '@typescript-eslint/no-use-before-define': 'off',
-        '@typescript-eslint/ban-types': 'off',
-        'require-await': 'off',
         'no-restricted-syntax': 'off',
         'no-await-in-loop': 'off',
-        'func-names': 'off',
+
+        // todo: maybe consider this rule globally?
+        quotes: ['error', 'single', { avoidEscape: true }],
```

Comment:
I believe this rule shouldn't be necessary because we have prettier and that should do this automatically. We rather should introduce some prettier check for CI for whole code base. prettier-eslint seems to miss some things anyway

Reply:
Ok, super, I just needed this for autofixing of backticks. Removing 29e9f6be8279856e6766466618a7e790fb802563